tests/integration/ha/test_ha.py

```diff
@@ -249,6 +252,61 @@ async def test_freeze_broker_with_topic_leader(
     assert_continuous_writes_consistency(result=result)
 
 
+@pytest.mark.abort_on_fail
+async def test_network_cut_without_ip_change(
+    ops_test: OpsTest,
+    c_writes: ContinuousWrites,
+    c_writes_runner: ContinuousWrites,
+):
+    # Let some time pass for messages to be produced
+    await asyncio.sleep(10)
+
+    topic_description = await get_topic_description(
+        ops_test=ops_test, topic=ContinuousWrites.TOPIC_NAME
+    )
+    initial_leader_num = topic_description.leader
+    leader_machine_name = await get_unit_machine_name(
+        ops_test=ops_test, unit_name=f"{APP_NAME}/{initial_leader_num}"
+    )
+
+    logger.info(
+        f"Throttling network for leader of topic '{ContinuousWrites.TOPIC_NAME}': {initial_leader_num}"
```

Comment:
**question:** What do we do to account for using the same topic in each test? In ZK we make use of Pytest's `request.info.name` fixture to get a unique zNode name for each test name. I'm guessing it doesn't matter as much here?

Reply:
I just delete the topic when the process is stopped. So the messages of each test remain unique